src/app/components/Footer/Link/index.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
```

Comment:
Fun fact with React 17 you shouldn't need to import React anymore, nothing needs to be changed in this PR though.